docs/data-management/qaqc-criteria-unprocessed.md

```diff
@@ -67,8 +70,10 @@ For accurate estimation of task activation, it is essential to check the quality
 
 !!! important "Our QC protocol is very lenient for task fMRI as those scans have been primarily acquired for QC purposes or to aid in methodological development, rather than holding scientific significance."
 
-- [ ] Exclude the QCT fMRI scan only in case of extreme distortion of the image. Potential causes can be e.g. failure in the image reconstruction, mistake in the header leading to the image being in the wrong orientation, extreme noise...
-- [ ] Exclude the BHT fMRI scan only in case of extreme distortion of the image. Potential causes can be e.g. failure in the image reconstruction, mistake in the header leading to the image being in the wrong orientation, extreme noise...
+- [ ] Exclude the QCT fMRI scan only in case of extreme distortion of the image. 
+Potential causes can be e.g. failure in the image reconstruction, mistake in the header leading to the image being in the wrong orientation, extreme noise...
```

Comment:
If the image has problems in the header, we need to check the conversion to NIfTI instead.

```suggestion
```